sklearn/ensemble/tests/test_stacking.py

```diff
@@ -223,10 +223,10 @@ def fit(self, X, y):
     "y, params, type_err, msg_err",
     [(y_iris,
       {'estimators': None},
-      ValueError, "Invalid 'estimators' attribute,"),
+      AttributeError, "Invalid 'estimators' attribute,"),
```

Comment:
I think we should keep ValueError here? AttributeError are reserved for other cases

Reply:
True, I should change the `Voting` estimators then.